gtd-timer/Attributes/ValidateModelAttribute.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using System;
+
+namespace gtdtimer.Attributes
+{
+    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Class)]
+    public class ValidateModelAttribute : ActionFilterAttribute
+    {
+        public override void OnActionExecuting(ActionExecutingContext context)
+        {
+            if (!context.ModelState.IsValid)
+            {
+                context.Result = new BadRequestObjectResult(context.ModelState);
```

Comment:
what happens if request is just 'null'?

Reply:
context.ModelState can't be null

Reply:
that's not what I meant. I mean if passed model is null  (postman request with no json but null)

Reply:
in other words - will Model.IsValid will be true, if 
context.ActionArguments?.FirstOrDefault().Value is null ?

Reply:
If passed model is null:
Model.IsValid is false
context.ActionArguments?.FirstOrDefault().Value is null